src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,112 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+	  	private String example = "";
+	    private String message_constraints = "";
+	    private String validation_regex = "";
+	    private String value = "";
+	    private boolean isPrivate;
+	    
+	    public Contact(String example, String message_constraints, String validation_regex, String value,
+		        boolean isPrivate) throws IllegalValueException {
+			super();
+			this.example = example;
+			this.message_constraints = message_constraints;
+			this.validation_regex = validation_regex;
+			this.isPrivate = isPrivate;
+			
+			String trimmedContact = value.trim();
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)